### DISCUSSION TITLE
Which keyboard shortcuts can be used in filter panel ?

### DISCUSSION THREAD

**Discussion by zxcat** [doublecmd/doublecmd#1284] [General]
Search/Filter helps do some things really fast with keyboard. But I cannot find, how to use it with keyboard only.

The only I found is switching mode between Filter (`cm_QuickFilter`) and Search (`cm_QuickSearch`). But there are more buttons on panel: match beginning/end, files/dirs, case sensitivity.

**Is there hotkeys for these buttons?**

**Comment by alexx2000** [comment]
No default hotkeys. But you can assign additional hotkeys to `cm_QuickSearch/cm_QuickFilter` with [parameters ](https://doublecmd.github.io/doc/en/cmds.html#cm_QuickSearch) (like `casesensitive=toggle`) to control this options.

**Reply by zxcat** [reply]
Great! Maybe show some mark in `Hot keys` > `Commands:` list near commands, which have parameters? It'll make things more clear.

**Reply by zxcat** [reply]
UI to set parameters would be great too, but simple mark (maybe with link to help) is enough at 1st

**Reply by zxcat** [reply]
Tried and it works fine, if either QuickFilter **or** QuickSearch used. But if I want to use **both** and switch between them then there is hot keys conflict. I can set the same key with `matchbeginning=toggle` in both Filter and Search, but it's normal behavior to same key do same action.
